### PR TITLE
Add how to enable 2FA

### PR DESCRIPTION
**Pull request by akegaviar**

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ module.exports = {
                     '/platform/view-your-organization-vault',
                     '/platform/view-activity-log',
                     '/platform/view-your-user-settings',
+                    '/platform/enable-two-factor-authentication',
```
Comment (neewy) [inline docs/.vuepress/config.js:62]: We have a https://docs.chainstack.com/platform/2fa link in our email templates: https://github.com/chainstack/email-templates/pull/165/files#diff-175525cb6cbbd9f1c45d5695d5db79ce68ecc991f8c1945d8f7b31d6c499824dR220. Should we change it here or in email-templates?

Comment (akegaviar) [reply docs/.vuepress/config.js:62]: That's something I had to comment on but didn't get around to. I suggest we remove the link from the email. I don't see what we can explain in there for a person who's already enabled 2FA.

Comment (neewy) [reply docs/.vuepress/config.js:62]: > I don't see what we can explain in there for a person who's already enabled 2FA.

Agree, let's take a look at GitHub:
Screenshot 2021-05-31 at 12 25 57

See https://docs.github.com/en/github/authenticating-to-github/securing-your-account-with-two-factor-authentication-2fa/about-two-factor-authentication. We should probably tell about recovery codes and reference it in the "enable 2fa" transactional email. WDYT?

**Review by neewy — APPROVED** [review]

File: docs/.vuepress/config.js (hunk 1 above)
Comment (easeev) [reply docs/.vuepress/config.js:62]: It looks like we have no docs to link from the email at the moment, so why do we link to docs from the email?

**Review by easeev — APPROVED** [review]
LGTM

**Pull request merged**